src/subtype.c

```diff
@@ -2583,6 +2585,46 @@ JL_DLLEXPORT int jl_type_morespecific_no_subtype(jl_value_t *a, jl_value_t *b)
     return type_morespecific_(a, b, 0, NULL);
 }
 
+JL_DLLEXPORT jl_svec_t *jl_match_method(jl_value_t *a, jl_value_t *b)
```

Comment:
This function is identical to `jl_env_from_type_intersection`; let's remove it. (`jl_env_from_type_intersection` looks like it's missing a GC root though)